Dockerfile

```diff
@@ -0,0 +1,46 @@
+
+# FIXME: try Alpine!!!!
+FROM openjdk:8u151-jre
+
+# Expose ports
+EXPOSE 8080
+
+# Add tags
+LABEL \
+  maintainer="The Petals team" \
+  contributors="Pierre Souquet (Linagora), Vincent Zurczak (Linagora)" \
+  github="https://github.com/petalslink"
+
+# Build arguments
+ARG COCKPIT_VERSION="LATEST"
+ENV DOWNLOAD_URL="https://gitlab.com/linagora/petals-cockpit/builds/artifacts/"
+
+# Copy the script (first)
+COPY docker-wrapper.sh /opt/petals-cockpit/docker-wrapper.sh
+
+# FIXME: unable to get the latest tag of gitlab without specific tag :
+# see https://gitlab.com/gitlab-org/gitlab-ce/issues/22273
+# and https://gitlab.com/gitlab-org/gitlab-ce/issues/26254
+# For now latest tag must be updated each release of cockpit.
+# Important: temporary system packages are deleted at the end.
+RUN apt-get update -y && \
+  apt-get install curl -y && \
+  apt-get upgrade -y 
+RUN if [ "${COCKPIT_VERSION}" = "LATEST" ]; then DOWNLOAD_URL="${DOWNLOAD_URL}v0.21.0/download?job=release-product"; \
```

Comment:
Try to replace it by a one-liner.
Example: `([ "${COCKPIT_VERSION}" == "LATEST" ] && DOWNLOAD_URL="") || ([ "${COCKPIT_VERSION}" == "SNAPSHOT" ] && DOWNLOAD_URL="")`

And then merge all the RUN instructions into a single one.

Reply:
Obviously, you should verify what I wrote works.
But something similar must be feasible.

Reply:
I changed it like you asked.